js/views/StatusMessage.js

```diff
@@ -11,6 +12,28 @@ export default class extends baseVw {
     this.listenTo(this.model, 'change', this.render);
   }
 
+  events() {
+    return {
+      'click [class^="js-"], [class*=" js-"]': 'onClick',
```

Comment:
The second one is prefaced with a space, so they are targeting two different things. For example, if the class attribute were `class="js-alpha sugar-time js-beta foo-bar-js-moo"`, the first one would target `js-alpha` whereas the second one would target `js-beta`.

If we just used the second one without the leading space, then it would correctly target the first one, but it would also target `foo-bar-js-moo`, which is not what we want.